spec/factories/disbursements.rb

```diff
@@ -8,7 +8,7 @@
     trait :valid do
       disbursement_date { age.days.ago.to_date }
       disbursement_type { DisbursementTypes.values.reject(&:other?).sample.to_s }
-      miles { 50 + (200**Random.new.rand(1.0).to_i) } # range: 50 -> 250
+      miles { 50 + (200**Random.new.rand(1.0).to_i) } # range: 50 -> 250 with logorithmic distribution
```

Comment:
I'm not entirely sure what you mean by logarithmic distribution here, but I don't think this code does what you think it does. `Random.new.rand(1.0)` is always less than 1, so `Random.new.rand(1.0).to_i` is always 0, so `(200**Random.new.rand(1.0).to_i)` is always 1, so this always sets miles to 51

Reply:
Ah, ok, I see what you mean by logarithmic distribution. You just need to move the `)` that's currently to the right of `to_i` the left of it.

Reply:
I don't understand why the wrote the rand function like this... I realised halfway through that the 1.0 broke things and thought I had removed it from everywhere... :(

fixed now

also for reference:
```
irb(main):001> Random.new.rand(1.0)
=> 0.9108281877297423
irb(main):002> rand(1.0)
=> 0
```